src/main/java/org/jboss/pnc/kafkastore/mapper/BuildStageRecordMapper.java

```diff
@@ -20,18 +20,32 @@
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
+import io.prometheus.client.Counter;
 import lombok.extern.slf4j.Slf4j;
+import org.eclipse.microprofile.metrics.MetricUnits;
+import org.eclipse.microprofile.metrics.annotation.Timed;
+import org.eclipse.microprofile.metrics.annotation.Gauge;
 import org.jboss.pnc.kafkastore.dto.ingest.KafkaMessageDTO;
 import org.jboss.pnc.kafkastore.model.BuildStageRecord;
 
 import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
 import java.io.IOException;
 import java.util.Optional;
 
 @ApplicationScoped
+@Timed
 @Slf4j
 public class BuildStageRecordMapper {
 
+    static final Counter exceptionsTotal = Counter.build()
```

Comment:
You can probably use a Gauge (2 Gauges) for these counters, according to examples in https://quarkus.io/guides/micrometer